docs/java-rest/high-level/apis/search.asciidoc

```diff
@@ -21,7 +21,7 @@ which contains everything that
 in the Rest API would be placed in the search request body.
```

Comment:
we are using REST here, so why "would be" , I think we can just say "which contains everything that is placed in the search request body"